webviz_subsurface/datainput/layeredmap/_layered_fence.py

```diff
@@ -0,0 +1,217 @@
+import numpy as np
+from ._image_processing import get_colormap, array_to_png
+from xtgeo import RegularSurface, Cube, Grid, GridProperty
+
+
+class LayeredFence:
+    '''### LayeredFence
+
+    Class to generate input for a LayeredMap component
+    to visualize subsurface data in a fence/cross-section perspective.
+    Xtgeo is used to slice regular surface grids along a given
+    polyline.
+
+
+    * `polyspec: 2D Numpy array for fence specification.
+
```

Comment:
- Add missing \` after polyspec. Specify what the columns/rows in the 2D `numpy` represents?
- Add description of `hinc`.